### ISSUE TITLE
Collaboration Cycle for Public Websites, Resources & Support, Landing Page v2 Enhancements

### ISSUE THREAD

**Issue by FranECross** [department-of-veterans-affairs/va.gov-team#95014]
### Verify information about your product or feature in the VFS Product Directory.

- [x] I have reviewed the [VFS Product Directory](https://depo-platform-documentation.scrollhelp.site/getting-started/vfs-product-directory) and confirm that my information is correct (or that I have taken appropriate actions to update or add my product information to the directory).

### VFS team name

Public Websites

### Product name

Resources & Support

### Feature name

Landing Page (including Search feature)

### GitHub label for team

Public Websites

### GitHub label for product

Resources and support

### GitHub label for feature

N/A

### Public DSVA Slack channel for VFS team

sitewide-public-websites

### When did/will you start working on this product/feature?

Research will be first, staring ~Nov 1st

### Will your work result in visible changes to the user experience?

Yes

### Are you doing research with VA.gov users?

Yes

### Will your work involve changes to...

Tools, applications and dynamic pages

### Does your work require non-trivial code changes or involve PII/PHI?

No

### Does your product/feature have Google Analytics tracking and a KPI dashboard in Domo?

Yes

### Do you need to capture any additional analytics or metrics?

No

### Will a VA editor (Drupal) notice this change?

Yes

### Product outline

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/campaign-landing-page/2024-Campaign%20Landing%20Page%20V2%20Enhancements%20Initiative.md

### Verify all relevant materials provided to Governance Team

- [x] I have provided all relevant and up-to-date links, screenshots, images, designs, etc. of the as-is version of the product

### Add the GitHub labels for your team, product, and feature to this ticket.

- [x] I acknowledge that I must add the GitHub labels for my team, product, and feature to this ticket.

**Comment by FranECross** [comment]
@shiragoodman Question about the order of touchpoints in the ticket. (I'm assuming they are in order of when we're supposed to complete them/schedule them?). For this ticket (and I would think for all tickets), is it possible to have the Research Review be just after Design Intent, which will place it before Architecture Intent. This is because our research sessions might introduce changes we want to make to the design, which would then impact the Architecture of the changes (or new app, if it was new). We won't have all the info for Architecture Intent until after the Research Sessions have completed and outcomes reviewed. TIA! cc @jilladams

**Comment by shiragoodman** [comment]
@FranECross yes, that's fine. The order of the touchpoints in this ticket does not indicate the order in which they should be scheduled. Most of the touchpoints are pretty fluid, so if it makes sense for you to engage in your Research Review now, then go for it. Please check out the [When do I schedule a Collaboration Cycle touchpoint?](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/overview#CollaborationCycleOverview-WhendoIscheduleaCollaborationCycletouchpoint?) section of the Collab Cycle Overview page for our scheduling recommendations. Hope this helps.

**Comment by aprocik1** [comment]
Note re. CAIA involvement: Danielle and Mikki are involved in this work, but the rest of our CAIA specialists likely won't be.

**Comment by FranECross** [comment]
@shiragoodman Thanks for the follow-up! We have resumed work on this project again, and we'll be scheduling a staging review sometime in the future. This project is competing with another one, so I think our staging review won't be for another month or so. Thanks again!
